src/common/transformations/src/transformations/transpose_sinking/ts_binary.cpp

```diff
@@ -30,6 +30,48 @@ TSBinaryForward::TSBinaryForward() : TSForwardBase() {
     transpose_sinking(matcher_name);
 }
 
+namespace {
+using NodePtr = std::shared_ptr<Node>;
+/**
+ * Inserts Unsqueeze node as a child to @arg node with axes {0, 2, ... N - 1}, where N = @arg n_dims
+ */
+NodePtr InsertBroadcastUnsqueezePReluSlope(const Output<Node>& node, size_t n_dims) {
```

Comment:
not so sure it works for arg rank = 1 and slope rank = 1 and arg_dim == slope_dim case

Reply:
I add unit test. It works